rocksdb_replicator/utils.h

```diff
@@ -24,6 +24,8 @@ const char* ReplicaRoleString(ReplicaRole role) {
         return "NOOP";
     case ReplicaRole::FOLLOWER:
         return "FOLLOWER";
+    case ReplicaRole::OBSERVER:
```

Comment:
can you add unit tests for this -- I think we already have a test for this function